src/service/pregnancyService.js

```
@@ -0,0 +1,397 @@
+    }
+  }
+
+  static async saveClinicalConservation(formObject) {
+    try {
+      const response = await axiosInstance.post(
+        `/parent/pregnancy/clinical`,
+        formObject
+      );
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)